client/src/notificationReceivers/errorManager.ts

```diff
@@ -97,14 +97,18 @@ export class ErrorManager implements Disposable, vscode.CodeActionProvider {
 
 	private _createDiagnostic(
 		range: vscode.Range,
-		message: string
+		error: PHPStanError
 	): vscode.Diagnostic {
-		const diagnostic = new vscode.Diagnostic(
-			range,
-			message,
-			vscode.DiagnosticSeverity.Error
-		);
+		const diagnostic = new vscode.Diagnostic(range, error.message);
 		diagnostic.source = 'PHPStan';
+		if (error.identifier) {
+			diagnostic.code = {
+				value: error.identifier,
+				target: vscode.Uri.parse(
+					`https://phpstan.org/error-identifiers/${error.identifier}`
```

Comment:
That's a good point, but maybe better to leave this possible-bug in because the positive from having it is a lot more positive than the negative gained from sometimes clicking a link that doesn't work imo.

Reply:
@ondrejmirtes Would you like to share your opinion on that? Maybe it would be possible to add optional URLs for error identifiers and return them in JSON formatter?

Reply:
PHPStan is not aware of what identifiers are present on the website and which are not.